Python/Automated_Noise_Removal/README.md

```diff
@@ -0,0 +1,9 @@
+OPEN-SOURCE SCRIPT FOR AUTOMATED NOISE REMOVAL USING PYTHON.
```

Comment:
Hey it would be great if you can follow the template we use.

Reply:
@vybhav72954 Just did! Please check and let me know if any further changes are required.
Have a good day!